### PR TITLE
[12.0][IMP] hr_timesheet_sheet: add kanban view

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@MiquelRForgeFlow this is primarily for activities, yet should not be in a separate module IMHO. And I guess this is the last improvement prior to v13 :D

File: hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml [hunk 1]
```diff
@@ -189,6 +189,67 @@
...
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="row no-gutters justify-content-end">
+                                <div class="col-2"><span class="text-muted">Period</span></div>
+                                <div class="col-8"><t t-esc="record.date_start.value"/> - <t t-esc="record.date_end.value"/></div>
+                                <div class="w-100"></div>
+                                <div class="col-2"><span class="text-muted">Total time</span></div>
+                                <div class="col-8"><t t-esc="record.total_time.value" t-options="{'widget': 'float_time'}"/></div>
```
Comment (MiquelRForgeFlow) [inline hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml]:
```suggestion
                                <div class="col-3"><span class="text-muted text-nowrap">Total time</span></div>
                                <div class="col-7"><t t-esc="record.total_time.value" t-options="{'widget': 'float_time'}"/></div>
```

Selection_109
Selection_110

File: hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml [hunk 2]
```diff
@@ -189,6 +189,67 @@
...
+                                                role="menuitem" class="dropdown-item">Set to Draft</a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="row no-gutters justify-content-end">
+                                <div class="col-3"><span class="text-muted">Period</span></div>
+                                <div class="col-7"><t t-esc="record.date_start.value"/> - <t t-esc="record.date_end.value"/></div>
+                                <div class="w-100"></div>
+                                <div class="col-3"><span class="text-muted">Total time</span></div>
```
Comment (MiquelRForgeFlow) [inline hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml]:
```suggestion
                                <div class="col-3"><span class="text-muted text-nowrap">Total time</span></div>
```
I put this before, but you may have not seen it due to the github indentation. Sorry.

Comment (alexey-pelykh) [reply hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml]: Thanks! Done for both lines

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
Fast-tracking as trivial to unblock v13 migration

**Pull request merged**